pkg/mutators/podtemplatespec.go

```diff
@@ -90,7 +91,7 @@ func (p *PodTemplateSpec) validateMemoryRequirements(ctx context.Context, contai
 	}
 
 	if limitRangeMemory.HasMaxLimitRequestRatio {
-		ratio := quantity.Div(*memoryLimit, *memoryRequest, infScaleMicro)
+		ratio := quantity.Div(*memoryLimit, *memoryRequest, infScaleMicro, inf.RoundUp)
```

Comment:
Use RoundUp for validation still as this prevents escapes to be rejected by LimitRanger.